onnxruntime/test/optimizer/resnet50_fusion_test.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include <vector>
+#include "core/graph/graph.h"
+#include "core/optimizer/conv_activation_fusion.h"
+#include "core/optimizer/conv_add_fusion.h"
+#include "core/optimizer/conv_add_act_fusion.h"
+#include "gtest/gtest.h"
+#include "graph_transform_test_builder.h"
+#include "test/test_environment.h"
+#include "test/util/include/asserts.h"
+
+namespace onnxruntime {
+namespace test {
+#if defined(ORT_RUN_EXTERNAL_ONNX_TESTS) && !defined(DISABLE_CONTRIB_OPS)
+class ResNet50FusionTests : public ::testing::Test {
```

Comment:
This is not an 'unit' test. This is more like an integration test. While this is valuable, it is not enough. Please add unit tests that target the kind of fusions that you added.